transactions/adjust-epoch-length/2024/sep-4/README.md

```diff
@@ -0,0 +1,45 @@
+# Adjust Epoch Length 
+
+This service account transaction updates the epoch length to increase the block rate,
+taking advantage of performance improvements. It also enables [FLIP204 - Smart-Contract-Specified Epoch Switchover Timing](https://github.com/onflow/flips/pull/204) on Mainnet.
+
+Currently, Mainnet runs at ~0.8 blocks/second. After this change, it will run at 1.25 blocks/second.
```

Comment:
```suggestion

### Context

Currently, Mainnet produces 0.8 blocks/second. Note that Flow's consensus does not run as fast as it can, but is slightly throttled, because we desire a consistent epoch switchover time every Wednesday at 19:00 UTC, so people know when to stake, bring up their nodes, and when they can expect to receive staking rewards. 

Epochs have a fixed length, measured in views. Though, if consensus proceeds at its maximum speed, the view rate of the network varies depending on network conditions (eg. load, # of available replicas, etc), resulting in a deviation of multiple hours per epoch, which accumulate over time. Hence, we have implemented an adaptive distributed control algorithm, called Cruise Control, which dynamically throttles the block production, so consensus progresses through a pre-defined number of views every second. 

Since the introduction of Cruise Control (April 2023), there have been a variety of performance improvements which improved the computing time for a consensus round (view) from roughly 830ms to 220ms. However, Cruse Control was still working with the setpoint of 0.8 blocks/second (see [our analysis](https://www.notion.so/flowfoundation/Cruise-Control-headroom-for-speedups-46dc17e07ae14462b03341e4432a907d?pvs=4) for more details).

### Goal

We reconfigured Curise Control, to allow consensus to proceed with a significantly faster block rate. This will help to reduce the latency for clients of the flow network (specifically Time To Finality, and Time To Executed). 
```